docs/counter.md

```diff
@@ -48,10 +49,10 @@ def Counter(label: String, initialStep: Int) =
       )
   }
 
-val app = div(
+val app: Resource[IO, HtmlDivElement[IO]] = div(
   h1("Let's count!"),
   Counter("Sheep", initialStep = 3)
 )
 
-app.renderInto(node.asInstanceOf[fs2.dom.Node[IO]]).allocated.unsafeRunAndForget()
+app.renderInto(node.asInstanceOf[fs2.dom.Node[IO]]).useForever.unsafeRunAndForget()
```

Comment:
This `asInstanceOf` looks a bit "hacky", at least for people not knowing background story. It also kind of leaks the opaque type. Maybe we can hide that somehow.

Reply:
Yeah, the `unsafeRunAndForget()` also is not so great 😅 maybe we can move these out of the code block 🤔